xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
-   xrefstyle="select:label"/>.
-  </para>
-  <example xml:id="ex-security-ldap-server-credentials-local">
-   <title>A <filename>.dsrc</filename> File for Local Administration</title>
-   <screen># cat ~/.dsrc
-[localhost]
-# Note that '/' is replaced with '%%2f'.
-uri = ldapi://%%2fvar%%2frun%%2fslapd-localhost.socket <co xml:id="co-ldap-server-dsrc-remote-ldapi"/>
-basedn = dc=example,dc=com
+uri = ldapi://<replaceable>%%2fvar%%2frun%%2fslapd-389ds-test.socket</replaceable> <co xml:id="co-ldap-server-dsrc-remote-ldapi"/>
+basedn = <replaceable>dc=test1,dc=com</replaceable>
 binddn = cn=Directory Manager
 </screen>
    <calloutlist>
+    <callout arearefs="co-ldap-server-dsrc-instance-name">
+      <para>
+        This must specify the instance name that you created in your
+        <filename>.inf</filename> file.
+      </para>
+    </callout>       
     <callout arearefs="co-ldap-server-dsrc-remote-ldapi">
       <para>
-       <!--Question: “When I use ldapi on the server that has the DS instance,
-       why don’t I need to provide my password?”--> When using <literal>ldapi</literal>
-       on the server where the &ds389; instance is running, your UID/GID will be
-       detected. If it is <literal>0/0</literal> (which means you are logged
-       in as &rootuser; user), the <literal>ldapi</literal> binds the local
-       &rootuser; as the directory server root dn (<literal>cn=Directory Manager</literal>)
-       of the instance. This allows local administration of the server, but also
-       allows you to set a machine-generated password for <literal>cn=Directory
-       Manager</literal> that no human knows. Whoever has administrator rights
-       on the server hosting the &ds389; instance can access the instance as
-       <literal>cn=Directory Manager</literal>.
+        <literal>ldapi</literal> detects the UID and GID of the user attempting 
+        to log in to the server. If the UID/GID are <literal>0/0</literal>,
+        <literal>ldapi</literal> binds the system root user (or sudo user) as 
+        the directory server root dn, which is 
+        <literal>cn=Directory Manager</literal>. If the UID/GID are not 
+        <literal>0/0</literal> then the user is not allowed to log into the server.
+      </para>
+      <para>
+        This has the benefit of protecting the ldap root password; the root user
+        can create the password, and then sudo users never need to know the 
+        password.
       </para>
+      <para>
+        In the URI the slashes are replaced with <literal>%%2f</literal>, so
+        in this example the path is 
+        <filename>/var/run/slapd-389/slapd-389ds-test.socket</filename>.
+      </para> 
     </callout>
    </calloutlist>
   </example>
+        <para>
+        You may also copy <filename>.dsrc</filename> to your home directory, for
+        running commands that do not need root privileges, such as listing 
+        users and groups:
```

Comment:
That probably hinges on ldapi autobind for uid != 0, or a different bind method.

Reply:
Adding to my todo-later list.